src/popup/modules/UserInterface.js

```diff
@@ -591,60 +629,109 @@ export function init() {
+                    // regenerate QR code
+                    setNewQrCodeSize(qrLastSize, true);
+                }
             }
-        }
 
-        // also set height of text (also to prevent display errors) when remember is enabled
-        if (qrCodeSize.sizeType === "remember" && qrCodeSize.hasOwnProperty("sizeText")) {
-            console.info("restore qr code text size:", qrCodeSize.sizeText);
-            // is saved as CSS string already
-            // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
-            qrCodeText.style.width = qrCodeSize.sizeText.width;
-
-            // detect too small size
-            const minimalSize = qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
-            if (window.innerHeight < minimalSize) {
-                console.error("too small size", window.innerHeight, "should be at least: ", minimalSize);
+            // also set height of text (also to prevent display errors) when remember is enabled
+            if (
+                qrCodeSize.sizeType === "remember" &&
+                qrCodeSize.hasOwnProperty("sizeText")
+            ) {
+                console.info("restore qr code text size:", qrCodeSize.sizeText);
+                // is saved as CSS string already
+                // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
+                qrCodeText.style.width = qrCodeSize.sizeText.width;
+
+                // detect too small size
+                const minimalSize =
+                    qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
+                if (window.innerHeight < minimalSize) {
+                    console.error(
+                        "too small size",
+                        window.innerHeight,
+                        "should be at least: ",
+                        minimalSize
+                    );
+                }
             }
         }
-    });
+    );
 
     const initQrTypespecificSettings = createContextMenu();
 
     // return Promise chain
-    return Promise.all([applyingMonospaceFont, applyingQrSize, applyingQrColor, initQrTypespecificSettings]);
+    return Promise.all([
+        applyingMonospaceFont,
+        applyingQrSize,
+        applyingQrColor,
+        initQrTypespecificSettings,
+    ]);
+}
+
+// zoom listener
+document.onkeydown = zoomQrCode;
+
+/**
+ * Listens to Alt + plus/minus keypresses to alter 
+ * QR code size.
+ *
+ * @function
+ * @returns {void}
+ */
+function zoomQrCode(e) {
+    var evtobj = window.event ? event : e;
+    if (evtobj.keyCode == 61 && evtobj.altKey && qrLastSize < 440) {
+        setNewQrCodeSize(qrLastSize + 30, true);
+
+        localStorage.setItem("lastSize", qrLastSize);
+    }
+    if (evtobj.keyCode == 173 && evtobj.altKey && qrLastSize > 50) {
```

Comment:
`keyCode` is kinda deprecated, see the warning at https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode, so let's better switch to alternatives, if possible.